app/src/main/java/com/pixlee/pixleeandroidsdk/gallery/ListViewHolder.java

```diff
@@ -1,11 +1,13 @@
-package com.pixlee.pixleeandroidsdk;
+package com.pixlee.pixleeandroidsdk.gallery;
 
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 import androidx.recyclerview.widget.RecyclerView;
 
+import com.pixlee.pixleeandroidsdk.R;
```

Comment:
where is `pixleeandroidsdk.R` being defined?  Can it be changed to something more descriptive then `R`?